sam_view.c

```diff
@@ -34,13 +34,21 @@ DEALINGS IN THE SOFTWARE.  */
 #include <stdbool.h>
 #include <assert.h>
 #include <getopt.h>
+#include <ctype.h>
 #include "htslib/sam.h"
 #include "htslib/faidx.h"
 #include "htslib/kstring.h"
 #include "htslib/khash.h"
 #include "htslib/thread_pool.h"
 #include "samtools.h"
 #include "sam_opts.h"
+
+#define DEFAULT_BARCODE_TAG "BC"
+#define DEFAULT_QUALITY_TAG "QT"
+
+#define xstr(s) str(s)
+#define str(s) #s
```

Comment:
These things are string literals already, so there's no need for the xstr trick.